pkgs/tools/compression/flips/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, gtk3, pkg-config, wrapGAppsHook }:
+
+stdenv.mkDerivation {
+  pname = "flips";
+  version = "1.31";
+
+  src = fetchFromGitHub {
+    owner = "Alcaro";
+    repo = "Flips";
+    rev = "24e6c77eb8d1ed6d0eb6da400c67ea229eca2278";
+    sha256 = "17b9mbpj59w6n5nin9g98rz4q9nz42m5wdir53jaw8cglz9xmsyp";
+  };
+
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
+  buildInputs = [ gtk3 ];
```

Comment:
Can we make it use libdivsufsort from nixpkgs?

Reply:
Probably, will look into that.

Reply:
Modified to use libdivsufsort.